bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4437,4 +4516,15 @@
             <column name="uuid" valueComputed=" uuid() "/>
         </insert>
     </changeSet>
+    <changeSet id="openmrs-core-201906201136" author="Samuel34">
```

Comment:
@samuelmale , I think you meant `bahmni-core-201906201136` right?

Reply:
I'm sorry I lost a little of context here? Did I say anything about this changeset? Btw sorry for the late response!

Reply:
@samuelmale,  I am referring to the line of the changeSet. The changeSet id is `openmrs-core-...` and I think you meant `bahmni-core-...`

Reply:
Nice catch man, just worked around it!